spec-main/api-net-spec.ts

```diff
@@ -1811,7 +1811,7 @@ describe('net module', () => {
       urlRequest.on('response', () => {});
       urlRequest.end();
       await delay(2000);
-      expect(numChunksSent).to.be.at.most(20);
+      expect(numChunksSent).to.be.at.most(25);
```

Comment:
i feel like i remember there being a good reason to set this at 20 but i don't remember what it was. fine bumping it up for now but maybe add a TODO for me?
```suggestion
      // TODO(nornagon): I think this ought to max out at 20, but in practice
      // it seems to exceed that sometimes. This is at 25 to avoid test flakes,
      // but we should investigate if there's actually something broken here and
      // if so fix it and reset this to max at 20, and if not then delete this
      // comment.
      expect(numChunksSent).to.be.at.most(25);
```